libwebauthn/src/ops/webauthn/get_assertion.rs

```diff
@@ -14,35 +25,190 @@ use crate::{
     webauthn::CtapError,
 };
 
-use super::{DowngradableRequest, SignRequest, UserVerificationRequirement};
+use super::{DowngradableRequest, RelyingPartyId, SignRequest, UserVerificationRequirement};
 
-#[derive(Debug, Default, Clone, Serialize)]
+pub const DEFAULT_TIMEOUT: Duration = Duration::from_secs(60);
+
+#[derive(Debug, Default, Clone, Serialize, PartialEq)]
 pub struct PRFValue {
     #[serde(with = "serde_bytes")]
     pub first: [u8; 32],
     #[serde(skip_serializing_if = "Option::is_none", with = "serde_bytes")]
     pub second: Option<[u8; 32]>,
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
 pub struct GetAssertionRequest {
     pub relying_party_id: String,
     pub hash: Vec<u8>,
     pub allow: Vec<Ctap2PublicKeyCredentialDescriptor>,
-    pub extensions: Option<GetAssertionRequestExtensions>,
+    pub extensions: GetAssertionRequestExtensions,
```

Comment:
I confess, I'm currently a bit lost in the details of this huge PR, but if I recall correctly, we used to use the `Option<>` here to decide if we have to include an extensions-map in the output. Some portals like demo.yubico.com are rather strict on what they expect, e.g. if they request extensions, they expect `extensions` in the response, even if the map is empty. And they do not want to see that empty map, if they didn't request one.
Is this still possible with this change?